tests/kernel/stack/stack/src/main.c

```diff
@@ -269,6 +296,69 @@ static void test_isr_stack_play(void)
 	k_thread_abort(tid);
 }
 
+/*the thread entry*/
```

Comment:
Should put space after /* and before */ (fix globally)

Reply:
OK, I have modified and re-push it.